README.md

```diff
@@ -17,22 +17,34 @@ Rideshare Programs are systems that help people organize carpooling/vanpooling.
 ## Objective
 We will look at data of drivers of a rideshare service and their ride history. We want to be able to see how many rides each driver has given and how much money a driver has made.
 
-Currently, the data is stored in a [CSV file](https://en.wikipedia.org/wiki/Comma-separated_values). In this assignment, you should take this data and thoughtfully organize it into a nested data structure that uses both arrays and hashes.
+Currently, the data is stored in a [CSV file](https://en.wikipedia.org/wiki/Comma-separated_values). In this assignment, you should take this data and thoughtfully organize it into **one single** var that is **one single** nested data structure. This nested data structure should use both arrays and hashes.
 
-## But why?
+### But why? Motivation to Organize Complex, Nested Data
 
-Given some data, a programmer often needs to be able to read, parse, and work with that data structure. Data that is organized into nested arrays and hashes can make this work easier. Ideally, data that is very well-organized will make this work easier to write, easier to read, and produce cleaner, higher-quality code.
+Given some data, a programmer often needs to be able to read, parse, and work with that data. Data that is organized into nested arrays and hashes can make this work easier. Ideally, data that is very well-organized will make this work easier to write, easier to read, and produce cleaner, higher-quality code.
 
 The most common way you'll see data represented is in [JSON](https://en.wikipedia.org/wiki/JSON). We'll learn more about JSON more in depth later, but what you need to know now is that data is often represented with several layers of nested data structures, ultimately making up one big data structure. If we practice working with nested data structures now, it will make working with JSON later easier!
 
 Have a look at [this example of JSON from Google Maps](sample_google_maps_data.json) _([Source](https://www.sitepoint.com/google-maps-json-file/))_ and answer the following questions.
+
+- What does this data represent?
 - How many different layers of data structures are there?
 - What data structures were used?
 - Why were those data structures used for those layers?
 
 ## Let's do this!
+
 Use the file `worksheet.rb` to complete this activity. Open up this file, and type your answers below each prompt. We will not be turning this assignment in, so feel free to write any comments or thoughts within this file!
 
+### Step 0: Understand the data we need to organize
+In your text editor, open up the file named `rides.csv`. It holds a pieces of concrete, instances of data. Answer the following questions, and write them down either in a text file, on paper, or on a whiteboard. If you're spending more than 5 minutes on a question, write down your questions and your ideas and move to the next question.
```

Comment:
Typo: get rid of the **a** in "It holds **a** pieces of concrete, instances of data."